mindmeld/models/taggers/lstm.py

```diff
@@ -447,6 +449,7 @@ def _define_optimizer_and_cost(self):
             log_likelihood, transition_params = tf.contrib.crf.crf_log_likelihood(
                 self.lstm_output_tf, flattened_labels, self.batch_sequence_lengths_tf
             )
+            del transition_params  # unused
```

Comment:
@vijay120, again is it correct that we aren't using these params?

Reply:
The issue is with the line above it, do this:
```
            flattened_labels = tf.cast(tf.argmax(self.label_tf, axis=2), tf.int32)
            log_likelihood, _ = tf.contrib.crf.crf_log_likelihood(
                self.lstm_output_tf, flattened_labels, self.batch_sequence_lengths_tf
            )
```

Reply:
Yeah, I intentionally didn't use underscore here until it was clear that we shouldn't be using them somewhere else